src/render/src/image.rs

```diff
@@ -21,26 +23,26 @@ pub struct Info {
     pub format: core::format::Format,
     pub memory: Memory,
 }
-/*
-impl Info {
-    /// Get image info for a given mip.
-    pub fn to_image_info(&self, mip: Level) -> NewImageInfo {
-        let (w, h, d, _) = self.kind.get_level_dimensions(mip);
-        ImageInfoCommon {
-            xoffset: 0,
-            yoffset: 0,
-            zoffset: 0,
-            width: w,
-            height: h,
-            depth: d,
-            format: (),
-            mipmap: mip,
-        }
-    }
 
-    /// Get the raw image info for a given mip.
-    pub fn to_raw_image_info(&self, mip: Level) -> RawImageInfo {
-        self.to_image_info(mip).convert(self.format)
+impl<B: Backend> handle::raw::Image<B> {
+    pub fn stable_state(&self) -> core::image::State {
```

Comment:
I think we should store this as a constant instead of recomputing

Reply:
A constant like a field in `Info`, or something more global ?

Reply:
yep, a field in `Info` would work